elasticdl/python/tests/allreduce_trainer_test.py

```diff
@@ -163,8 +163,8 @@ def test_elastic_run(self):
         controller.set_broadcast_model(model)
         controller.set_broadcast_optimizer(optimizer)
         controller.broadcast()
-        result = controller.train_one_batch_with_retries(self.train)
         controller.restore()
+        result = controller.train_one_batch_with_retries(self.train)
```

Comment:
Why do we move `train_one_batch_with_retries` after `restore`?

Reply:
This is a mistake in the PR and I will remove the modification.